src/main/resources/swagger/api-docs.yaml

```diff
@@ -1236,6 +1236,70 @@ paths:
         '200':
           description: Successful Request
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/catalog:
+    get:
+      responses:
+        200:
+          description: Successful Request
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/WorkspaceCatalog'
+        403:
+          description: Insufficient permissions to get workspace catalog permission (must be owner or admin)
+          schema:
+            $ref: '#/definitions/ErrorReport'
```

Comment:
Yes, for the get we will return a 404, but for the patch, we will return a 403 if they have read or write but not owner or admin.